technologies/app/jupyspark/metadata.yaml

```diff
@@ -0,0 +1,30 @@
+version: v1
+type: APP
+id: jupyspark
+label: "[Lab] Jupyter Notebook with Spark"
+baseline: Data Science Notebook
+description: "The Jupyter Notebook is an open-source web application that allows you to create and share documents that contain live code, equations, visualization and narrative text. Uses include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning and much more."
+available: true
+icon: jupyter
+backgroundColor: "#E87A35"
+customFlags: []
+contexts:
```

Comment:
Could we add the information that this notebook is running with Python 3.6 and Spark 2.4.5? Useful for the users

Reply:
OK I'll do that ... 
Moreover the id should be updated to reflect those version isn't it ?

Reply:
yes id should be also updated